pkg/blockstorage/awsebs/awsebs.go

```diff
@@ -82,7 +83,7 @@ func newEC2Client(awsRegion string, config *aws.Config) (*EC2, error) {
 	}
 	s, err := session.NewSession(config)
 	if err != nil {
-		return nil, errors.Wrap(err, "Failed to create session for EFS")
+		return nil, errors.Wrap(err, "Failed to create session for EBS")
```

Comment:
Why "EBS"? Why not "Failed to create EC2 session"?

Reply:
I just fixed the some copy paste error in the past. It definitely shouldn't say EFS

Reply:
This is getting the Ec2client for the EBS provider.